app/api/rest_api.py

```diff
@@ -43,7 +43,7 @@ async def enable(self):
         self.app_svc.application.router.add_route('*', '/api/rest', self.rest_core)
         self.app_svc.application.router.add_route('GET', '/api/{index}', self.rest_core_info)
         self.app_svc.application.router.add_route('GET', '/file/download_exfil', self.download_exfil_file)
-        self.app_svc.application.router.add_route('GET', '/{tail:(?!plugin/).*}', self.handle_catch)
+        self.app_svc.application.router.add_route('GET', '/{tail:(?!plugin|api/v2/).*}', self.handle_catch)
```

Comment:
```suggestion
        self.app_svc.application.router.add_route('GET', '/{tail:(?!plugin/|api/v2/).*}', self.handle_catch)
```
To maintain the previous path condition

Reply:
Thanks.
I fixed up the original commit with the `plugin` to  `plugin/` change.

However, I put the `plugin/` to `plugins/` into a dedicated commit.

I saw a lot of `/plugin/...` endpoints (global RegEx search: `add_route.+plugin/`).
Are you sure the change that adds an `s` is correct?

Reply:
No, good catch. I was going off the URLs in my browser when clicking different plugins, but the ones in `add_route` don't have the `s` as you pointed out. We'll just do `plugin/`

Reply:
OK. I removed the commit and rebased my branch on the new `master`.